### DISCUSSION TITLE
Distrobox with a different Nvidia driver version

### DISCUSSION THREAD

**Discussion by doganulus** [89luca89/distrobox#1238] [General]
Some applications require a specific version of Nvidia drivers. 

Can I install Nvidia drivers inside a distro (which is different than the host's driver version) and make GPU work?

**Comment by 89luca89** [comment]
Yep, just don't use the `--nvidia` flag, and install your driver in the container

**Reply by doganulus** [reply]
I am getting the following error when installing the driver on a host with a different driver version. No `--nvidia` flag is used when creating the (debian12) distrobox.

> Mismatching nvidia kernel module loaded                                                         
>                                                                                                  
> The NVIDIA driver that is being installed (version 525.147.05) does not match the nvidia
> kernel module currently loaded (version 545.29.06).           
>                                                                                              
> The X server, OpenGL, and GPGPU applications may not work properly.                              
>
> The easiest way to fix this is to reboot the machine once the installation has finished. You
> can also stop the X server (usually by stopping the login manager, e.g. gdm3, sddm, or xdm),
> manually unload the module ("modprobe -r nvidia"), and restart the X server.

Do you suggest any flag to ensure the driver isolation or any other distrobox feature for this use case?
